packages/@aws-cdk-testing/cli-integ/resources/templates/simpleDDB.yaml

```diff
@@ -0,0 +1,79 @@
+AWSTemplateFormatVersion: "2010-09-09"
```

Comment:
Where is this being used?

Reply:
It's not as it doesn't pass for csharp. Had planned to have multiple integ test files for the various languages but these aren't being used until csharp passes generally